app/views/employment/CheckEmploymentDetailsView.scala.html

```diff
@@ -31,54 +28,91 @@
         govukSummaryList: GovukSummaryList,
         govukInsetText: GovukInsetText
 )
-@(employerNameData: String, employerRefData: Option[String], employmentDetails: Option[EmploymentData], taxYear: Int)(implicit user: User[AnyContent], messages: Messages, appConfig: AppConfig)
+@(employerNameData: String, employerRefData: Option[String], employmentDetails: Option[EmploymentData], taxYear: Int, isInYear:Boolean, employmentId:String, isCustomerData:Boolean)(implicit user: User[AnyContent], messages: Messages, appConfig: AppConfig)
 
 
 @headingForUse = @{
     messages(s"employment.employmentDetails.heading.${if(user.isAgent) "agent" else "individual"}")
 }
 
+@margin = @{
+    if(!isCustomerData) "govuk-!-margin-bottom-2" else "govuk-!-margin-bottom-4"
```

Comment:
are the difference in margins needed to match the prototype?

Reply:
I couldn't find an example in the prototype where there wasn't paragraph text. For when there wasn't <p> text margin-2 did look kind of ridiculous though